package.json

```diff
@@ -33,6 +33,7 @@
     "grunt-phpunit": "^0.3.6",
     "grunt-wp-readme-to-markdown": "^2.0.1",
     "load-grunt-tasks": "^5.1.0",
+    "mochawesome-json-to-md": "^0.7.2",
```

Comment:
Assuming this was added on purposed?

Reply:
Yeah, the newest version of this dependency causes our E2E tests to show as failed because the summary can't be updated correctly. See https://github.com/10up/eight-day-week/actions/runs/8084407853/job/22089631355. We've been downgrading this to the prior version across our repos, though would be great to figure out if there's a better work around that allows us to run the newest version